docs/automated-testing/test-planning.md

```diff
@@ -0,0 +1,219 @@
+# Test planning
+
+We should be intentional when we think about how to test our applications. One way to make sure that we are testing the right things is to build test plans for various scenarios and work out test cases at the design stage of a user story.
+
+This content aims to show some ways we can think about planning and give some examples using a web site with a login portal as an example.
+
+## Building test cases for a user story
+
+As we design the code for a user story we can start by defining a set of test cases that will ensure that all acceptance criteria are met.
+
+This can be done as an exercise that includes both the developers and other stakeholders. Some of the test cases may initially be manual, and some may be planned for automation, depending on the situation.
+
+Going through this exercise doesn't only produce test cases, it also helps clarify the acceptance criteria and informs how we should build the solution.
+
+1. Understand the Acceptance Criteria
+    - Thoroughly read and understand the acceptance criteria for the story
+    - Clarify any ambiguities with the product owner or stake holders
+
+1. Identify Test Scenarios
```

Comment:
nit: seems like numbering is not going in order.

Reply:
good catch, will fix